static/styles/recent_topics.css

```diff
@@ -131,6 +144,18 @@
             align-self: center;
         }
 
+        .unread_count_pm {
+            margin-right: 46px;
```

Comment:
Same question with this 46px; I don't understand how it related to the 53px below either.